src/app/Upload.svelte

```diff
@@ -100,6 +105,16 @@
     color: var(--blue);
     padding-bottom: 1em;
   }
+
+  .download-link {
+    display: block;
+    margin-top: 1.5em;
+  }
+
+  .download-link--disabled {
+    color: #666;
```

Comment:
I would extract this color into the global style since disabled color will be potentially used in multiple places

Reply:
already changed :)